src/components/User/User.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './User.scss';
+
+export function User({ user }) {
```

Comment:
```suggestion
export function User({ name }) {
```
Does object `User` has the `user` property? It has `name` property, which you should have used here